camel/storages/vectordb_storages/milvus.py

```diff
@@ -0,0 +1,360 @@
+            "metric_type": "COSINE",
+            "index_type": "AUTOINDEX",
+            "params": {},
+        }
+
+        fields = [
+            FieldSchema(name='id', dtype=DataType.VARCHAR,
+                        descrition='A unique identifier for the vector',
+                        is_primary=True, auto_id=False, max_length=65535),
+            FieldSchema(
+                name='payload', dtype=DataType.VARCHAR,
+                description=('Any additional metadata or information related'
+                             'to the vector'), max_length=65535),
+            FieldSchema(
+                name='vector', dtype=DataType.FLOAT_VECTOR,
+                description='The numerical representation of the vector',
+                dim=self.vector_dim)
+        ]
+        schema = CollectionSchema(fields=fields,
+                                  description='Collection schema')
+
+        self._client.create_collection_with_schema(
+            collection_name=collection_name,
+            schema=schema,
+            index_params=index_params,
+            **kwargs,
+        )
+
+    def _delete_collection(
+        self,
+        collection_name: str,
+    ) -> None:
+        r"""Deletes an existing collection from the database.
+
+        Args:
+            collection (str): Name of the collection to be deleted.
+        """
+        self._client.drop_collection(collection_name=collection_name)
+
+    def _collection_exists(self, collection_name: str) -> bool:
+        r"""Checks whether a collection with the specified name exists in the
+        database.
+
+        Args:
+            collection_name (str): The name of the collection to check.
+
+        Returns:
+            bool: True if the collection exists, False otherwise.
+        """
+        for c in self._client.list_collections():
+            if collection_name == c:
+                return True
+        return False
+
+    def _generate_collection_name(self) -> str:
+        r"""Generates a unique name for a new collection based on the current
+        timestamp.
+
+        Returns:
+            str: A unique collection name.
+        """
+        return datetime.now().isoformat()
+
+    def _get_collection_info(self, collection_name: str) -> Dict[str, Any]:
+        r"""Retrieves details of an existing collection.
+
+        Args:
+            collection_name (str): Name of the collection to be checked.
+
+        Returns:
+            Dict[str, Any]: A dictionary containing details about the
+                collection.
+        """
+        vector_count = self._client.num_entities(collection_name)
+
+        collection_info = self._client.describe_collection(collection_name)
+        collection_id = collection_info['collection_id']
+        dim_value = next(
+            (field['params']['dim']
+             for field in collection_info['fields'] if field['description'] ==
+             'The numerical representation of the vector'), None)
```

Comment:
filter by `field['type'] == 101`:
 refer to https://milvus.io/api-reference/pymilvus/v2.3.x/MilvusClient/Collections/DataType.md

Reply:
here I used the description rather than type code to make my code more easy to understand, do you think it's fine if I keep the description rather than use type code?